samples/client/petstore/typescript-angular2/default/api/PetApi.ts

```diff
@@ -1,9 +1,9 @@
 /**
  * Swagger Petstore
- * This is a sample server Petstore server.  You can find out more about Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).  For this sample, you can use the api key `special-key` to test the authorization filters.
+ * This is a sample server Petstore server.  You can find out more about Swagger at <a href=\"http://swagger.io\">http://swagger.io</a> or on irc.freenode.net, #swagger.  For this sample, you can use the api key \"special-key\" to test the authorization filters
  *
  * OpenAPI spec version: 1.0.0
- * Contact: apiteam@swagger.io
+ * Contact: apiteam@wordnik.com
```

Comment:
The latest master is using `petstore.yaml` instead of `petstore.json` (which still references wordnik.com) so rebasing the PR on the latest master should avoid the above change.